conf/server/server_full.conf

```diff
@@ -65,26 +65,18 @@ server {
         }
 
         # federates_with "<trust domain>": configures the address of a bundle endpoint used to
-        # get a trust bundle for "<trust domain>". This section can be repeated per trust domain.
+        # get a trust bundle for "<trust domain>". This section must be repeated for each
+        # federated trust domain.
         federates_with "domain1.test" {
-            # bundle_endpoint: configuration of other server's bundle endpoint.
-            bundle_endpoint {
-                # address: IP or DNS name of the bundle endpoint that provides the trust
-                # bundle to federate with `"<trust domain>"`.
-                address = "1.2.3.4"
-
-                # port: Port number of the bundle endpoint. Default: 443
-                # port = 443
-
-                # spiffe_id: Expected SPIFFE ID of the bundle endpoint server. This
-                # is ignored if use_web_pki is true. Default: SPIRE Server SPIFFE ID
-                # within the `"<trust domain>"`.
-                # spiffe_id = ""
-
-                # use_web_pki: If true, indicates that this server must use Web PKI to
-                # authenticate the bundle endpoint, otherwise SPIFFE authentication is used.
-                # Default: false.
-                # use_web_pki = false
+            # bundle_endpoint_url: Bundle endpoint URL. Default: "".
+            bundle_endpoint_url = "https://example.com/global/bundle.json"
+
+            # bundle_endpoint_profile "<https_web|https_spiffe>". Endpoint profile.
+            bundle_endpoint_profile "https_spiffe" {
```

Comment:
nit: i think spacing is off for this federates_with block

Reply:
Would be great to include an example of `https_web` profile, we can update comments here that refer to it as well